lib/node/rpc.js

```diff
@@ -2609,7 +2609,7 @@ class RPC extends RPCBase {
     } catch (err) {
       if (err.type === 'VerifyError') {
         this.logger.warning('RPC block rejected: %s (%s).',
-          block.hash(), err.reason);
+          block.toHex(), err.reason);
```

Comment:
I think the convention around the code is to use a `%x` placeholder for hash->string. That would make this port to bcoin as well (where block does not extend bufio)